### DISCUSSION TITLE
Improving computational time

### DISCUSSION THREAD

**Discussion by VitorLuisAndrade** [ImperialCollegeLondon/sharpy#178] [Q&A]
I am using SHARPy for a graduation project. I am using the UVLM to simulate a coupled model for the structure and aerodynamics of a wind turbine. However, simulating only a couple of minutes takes days on a reasonable powerful computer. Is there anyway to accelerate and speed up the simulation? I appreciate the help.

**Comment by ArturoMS13** [comment]
Dear VitorLuisAndrade, 

You are right that converged simulations of wind turbines are computationally expensive. My advice is, first, to run mesh convergence analysis to reduce the discretisation to the minimum (including time step). Second, try the new wake discretisation options in the develop branch  (https://github.com/ImperialCollegeLondon/sharpy/pull/124 and sharpy/tests/uvlm/dynamic/test_wake_cfl_n1.py) that have been improved and clarified in the branch dev_blade_pitch_v2. Finally, if you are working at Imperial College London (or other institution), SHARPy can be run in computer clusters such as Imperial College HPC (https://www.imperial.ac.uk/admin-services/ict/self-service/research-support/rcs/)

I hope this helps. Best regards, Arturo.
